lib/ui/painting/codec.cc

```diff
@@ -371,20 +371,16 @@ MultiFrameCodec::MultiFrameCodec(std::unique_ptr<SkCodec> codec,
   compressedSizeBytes_ = codec_->getInfo().computeMinByteSize();
```

Comment:
clang-format is controlling the indentation here. We can set it off for certain lines if we want to tweak it though. I'm going to leave this as-is for now since it's not related to this PR, can alter this in a followup.